### PR TITLE
More testing and slight changes to the help cli

### PR DESCRIPTION
**Pull request by ductiletoaster**
working through updating help and adding validation but wanted to close the gap even further on these tests first.

### CODE REVIEW

File: packages/pixeloven/cli/src/commands/compile.test.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe("@pixeloven/cli", () => {
                 Stub.system.spawn.resolves({
                     status: 0,
                 });
-                const context = await cli.run("compile ts ./some/file.ts");
+                const context = await cli.run("compile tsx ./some/file.ts");
```
Comment (jimmynotjim) [inline packages/pixeloven/cli/src/commands/compile.test.ts]: but it's a ts file?

File: packages/pixeloven/cli/src/commands/lint.test.ts [hunk 2]
```diff
@@ -92,12 +92,12 @@ describe("@pixeloven/cli", () => {
...
                 expect(Stub.process.exit.calledWithExactly(0)).toEqual(true);
             });
-            it("should succeed linting ts/tsx with stylelint.json", async () => {
+            it("should succeed linting css/scss with stylelint.json", async () => {
                 Mock.core.expects("resolvePath").returns(true);
                 Stub.system.spawn.resolves({
                     status: 0,
                 });
-                const context = await cli.run("lint scss");
+                const context = await cli.run("lint css");
```
Comment (jimmynotjim) [inline packages/pixeloven/cli/src/commands/lint.test.ts]: Oh wait, are you reducing the task arguments to `css` and `tsx`? If we're going to do that, can we do `styles` and `scripts`?

File: packages/pixeloven/cli/src/commands/compile.test.ts (hunk 1 above)
Comment (ductiletoaster) [reply packages/pixeloven/cli/src/commands/compile.test.ts]: They are just aliases for each other. This was just to get full coverage without needing to write an entire new test to cover one extra branch.

File: packages/pixeloven/cli/src/commands/lint.test.ts (hunk 2 above)
Comment (ductiletoaster) [reply packages/pixeloven/cli/src/commands/lint.test.ts]: Negative, same as the above comment was a simple way to get more coverage without needing a new test for an alias

**Pull request merged**